src/api/Feed.js

```diff
@@ -199,11 +199,24 @@ class Feed extends Base {
                 }
             },
             (e: ErrorResponseData, code: string) => {
-                this.feedErrorCallback(true, e, code);
+                this.updateAnnotationErrorCallback(e, code, annotationId);
```

Comment:
I don't think this new method is necessary, since it's still wrapped in an anonymous function. Can we just add the `this.updateFeedItem` line above?

Reply:
Yep will do